client/src/components/Home.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default function Home() {
+    return (
+        <div>
+            
+        </div>
+    )
+}
```

Comment:
Files like this cause bloat to your code, we can remove these to keep things more organized through your project